src/app/containers/MediaPlayer/index.jsx

```diff
@@ -70,6 +70,11 @@ const MediaPlayerContainer = ({
       aresMediaBlock,
     ),
     type: format === 'audio' ? 'audio' : 'video',
+    guidanceMessage: pathOr(
+      null,
+      ['model', 'blocks', 0, 'model', 'versions', 0, 'warnings', 'short'],
+      aresMediaBlock,
+    ),
```

Comment:
I would expect see unit tests for this, no?

Reply:
No unit test is required for this, as we are just adding a arg to an already existing prop ie mediaInfo, and is covered for by this test https://github.com/bbc/simorgh/blob/cfb3deff62c0b3293376d82493a9fecdd53a2bd4/src/app/containers/MediaPlayer/index.test.jsx#L12 whose snapshot checks that guidance message is also available from the  data here https://github.com/bbc/simorgh/blob/cfb3deff62c0b3293376d82493a9fecdd53a2bd4/src/app/containers/MediaPlayer/fixtureData.jsx#L44.

e2e tests should suffice.

Reply:
What about tests that describe the output when there is no guidance?

Reply:
Would be covered in the e2e, its part of the discussion that @jamesbrumpton was talking about, about having a conditional in #4590

Reply:
As discussed we're going to make a separate issue to revisit the unit tests.